packages/styles/src/elements/_index.scss

```
@@ -2,6 +2,7 @@
 
 @use 'body';
 @use 'anchor';
+@use 'bullet-list';
```

Could be named `list-bullet` as well just to place it with all the other list types.

```suggestion
@use 'list-bullet';
```